### PR TITLE
SPCTR-3480: Fix - Container gradient overlay not working as intended.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Container background overlay gradient color has no on change of colors when picked different colors than default .
https://brainstormforce.atlassian.net/browse/SPCTR-3480

### Screenshots

https://d.pr/v/1aRfdv

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/background/index.js [hunk 1]
```diff
@@ -922,7 +922,7 @@ const Background = ( props ) => {
 								<>
 									<div className="uag-background-image-overlay-gradient">
 										<GradientSettings
-											backgroundGradient={ props.backgroundOverlayGradient }
+											backgroundGradient={ props.backgroundGradient }
```
Comment (bsf-zanev) [inline src/components/background/index.js:925]: Is this backward compatible?

Do the users that are using `backgroundOverlayGradient` lose all their changes?

If so, we may have to handle `backgroundOverlayGradient` migration.

**Review by bsf-zanev — COMMENTED** [review]
Had one concern.

File: src/components/background/index.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/components/background/index.js:925]: Actually we are trying to reuse the gradient component for overlay gradient as well, but there was a mistake made that the backgroundOverlayGradient variable was used where it should be backgroundGradient so the value was not update and only the default value was being applied, we can discuss this on call for more clarity.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**